kieserver/63/src/test/java/org/jboss/test/arquillian/ce/decisionserver/DecisionServerMultiContainerAmqTest.java

```diff
@@ -69,7 +69,7 @@ public void testSecondDecisionServerContainer() throws Exception {
     }
 
     @Test
-    public void testDecisionServerSecondContainerAMQ() throws NamingException {
+    public void testDecisionServerSecondContainerAMQ() throws Exception {
```

Comment:
It seems that throwing a generic Exception is not a good practice in Java. Is there a strong reason for that?